lib/internal/modules/cjs/loader.js

```diff
@@ -451,15 +451,17 @@ function tryPackage(requestPath, exts, isMain, originalPath) {
       err.code = 'MODULE_NOT_FOUND';
       err.path = path.resolve(requestPath, 'package.json');
       err.requestPath = originalPath;
-      // TODO(BridgeAR): Add the requireStack as well.
+
```

Comment:
The `requireStack` must be added to this error. That does not seem to be addressed here?